### DISCUSSION TITLE
VFL

### DISCUSSION THREAD

**Discussion by NastyNastya2024** [dmlc/xgboost#11309] [Q&A] [answered]
In the context of vertical federated learning, once the passive side receives the Hessians and gradients, how is the training process conducted on that side? If possible, could you kindly provide code examples illustrating where this occurs and which algorithm is used?

**Comment by trivialfis** [comment]
Could you please see the proposal here: https://github.com/dmlc/xgboost/issues/8424 .

**Reply by NastyNastya2024** [reply]
Thank you for your responce! Could you when the vfl branch was merged into nvflyer, it is currently unclear how to use this library. This In which version of XGBoost will this be released - 3.000?

**Reply by trivialfis** [reply, marked as answer]
2.1 works as well, just install with `pip` from PyPI as normal, but I haven't tracked the changes in nvflare, I remember they have some demos.
